### ISSUE TITLE
Maven artifacts are inaccessible

### ISSUE THREAD

**Issue by yalchinAlv** [gitblit-org/gitblit#1449]
Is this project still alive?

If yes, is it already known or expected that the jar artifacts on Maven is not accessible?
https://mvnrepository.com/artifact/com.gitblit/gitblit/1.9.3
https://gitblit.github.io/gitblit-maven/com/gitblit/gitblit/1.9.3/gitblit-1.9.3.jar - Returns 404

Seems like https://gitblit.github.io/gitblit-maven/ is down.

**Comment by flaix** [comment]
Yes and no. Yes, it is expected that https://gitblit.github.io/gitblit-maven/ is down. Gitblit was moved to an organisation which unfortunately runs under a different name, `gitblit-org.github.io`. No, I was not aware that mvnrepository listed the gitblit.github.io/gitblit-maven repo at all. Ideally that could be changed on mvnrepository.

I don't know if it is possible to create a redirect site on Github for the old URL.
